clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeliverExecutor.scala

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.clio.client.dispatch
+
+import akka.http.scaladsl.model.HttpResponse
+import akka.http.scaladsl.model.headers.HttpCredentials
+import org.broadinstitute.clio.client.commands.{DeliverWgsCram, MoveWgsCram}
+import org.broadinstitute.clio.client.util.IoUtil
+import org.broadinstitute.clio.client.webclient.ClioWebClient
+import org.broadinstitute.clio.transfer.model.WgsCramIndex
+import org.broadinstitute.clio.transfer.model.wgscram.TransferWgsCramV1Metadata
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * Special-purpose CLP for delivering crams to Firecloud workspaces.
+  *
+  * Wraps the move CLP with extra IO / upsert logic.
+  */
+class DeliverExecutor(deliverCommand: DeliverWgsCram) extends Executor {
```

Comment:
Can this be written so it will work for any index? If not, should the name be changed to reflect that it's specific to wgs cram?

Reply:
It's special-cased to crams, will rename the class.